tests/e2e/defaults/micronaut-core.ini

```diff
@@ -0,0 +1,10 @@
+# Copyright (c) 2022 - 2024, Oracle and/or its affiliates. All rights reserved.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl/.
+
+[analysis.checks]
+# By default, we don't exclude any checks.
```

Comment:
Maybe a note about why these checks are being excluded would be better here instead of this.

Reply:
See commit [6275820](https://github.com/oracle/macaron/pull/676/commits/627582061efd0d22991e34e3a3f38e61f524c213)